gems/aws-sdk-route53/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 Unreleased Changes
 ------------------
 
+1.11.0 (2018-06-26)
```

Comment:
Ah, you don't need to provide the version stuff here, our release tool will bump the version automatically.
Sorry for the confusion, could you remove these 3 lines and just keep the issue entry with its following new line?
After this is cleaned up, I'll merge the PR